dist/leaflet.distortableimage.css

```diff
@@ -40,3 +40,16 @@
 img.leaflet-image-layer.selected {
   box-shadow: 0px 0px 0px 12px #ffea00;
 }
+
```

Comment:
Is this for this PR? Or is it from another, maybe? Thanks!

Reply:
@jywarren, These are for the CSS manipulations for the popup menu I mentioned above.